projects/kubernetes/kubernetes/Makefile

```diff
@@ -100,6 +110,7 @@ cleanup-artifacts:
 	rm -rf $(OUTPUT_DIR)/{attribution,LICENSES,pause} $(OUTPUT_DIR)/bin/{LICENSES,ATTRIBUTION.txt}
 	mv $(OUTPUT_DIR)/bin/* $(ARTIFACTS_PATH)/bin
 	cp $(OUTPUT_DIR)/ATTRIBUTION.txt $(ARTIFACTS_PATH)
+	if [ "$(FAKE_ARM_IMAGES_FOR_VALIDATION)" = "true" ]; then rm -rf $(OUTPUT_DIR)/images/bin/linux/arm64; fi
```

Comment:
is `FAKE_ARM_IMAGES_FOR_VALIDATION` meant to be set somewhere?

Reply:
i see it here https://github.com/aws/eks-distro/blob/2f71476c12d964cdd2f249cd3ceb76442b93aee0/Common.mk#L414